zproject/settings.py

```diff
@@ -687,6 +687,11 @@ def get_dirs(self):
 RATE_LIMITING_RULES = [
     (60, 200),  # 200 requests max every minute
 ]
+
+RATE_LIMITING_MIRROR_REALM_RULES = [
+    (300, 50),  # 50 emails max (per realm) every 5 minutes
+]
```

Comment:
Given that this is a realm-level rule, we may want something more like these slightly higher defaults:
```
(60, 50),
(300, 120),
(3600, 600)
```

(600 / hour, but no bursts about 50 in a minute or 120 in 5 minutes).